File: result/translate/starchat/java2cpp/c++/FIND_SUM_EVEN_INDEX_BINOMIAL_COEFFICIENTS.cpp
int f_gold ( int n ) {
    int **C = new int *[n + 1];
    for (int i = 0; i <= n; i++) C[i] = new int[n + 1];
    for (int i = 0; i <= n; i++) {
        for (int j = 0; j <= min(i, n); j++) {
            if (j == 0 || j == i) C[i][j] = 1;
            else C[i][j] = C[i - 1][j - 1] + C[i - 1][j];
        }
    }
    int sum = 0;
    for (int i = 0; i <= n; i += 2) sum += C[n][i];
    delete[] C;
    return sum;
}

File: result/translate/starchat/java2python/python/FIND_SUM_EVEN_INDEX_BINOMIAL_COEFFICIENTS.py
def f_gold(n):
        C = [[0 for _ in range(n + 1)] for _ in range(n + 1)]
        for i in range(n + 1):
            for j in range(n + 1):
                if j == 0 or j == i:
                    C[i][j] = 1
                else:
                    C[i][j] = C[i - 1][j - 1] + C[i - 1][j]
        sum = 0
        for i in range(0, n, 2):
            sum += C[n][i]
        return sum